pkgdown/extra.css

```diff
@@ -1,31 +1,34 @@
-.navbar-inverse {
-    background-color: #0099f9;	
+.navbar {
+    background-color: rgb(78,196,215) !important;
```

Comment:
1. It would be pretty nice to use [bslib variables](https://pkgdown.r-lib.org/articles/customise.html#bslib-variables) instead of `!important` to customize the colors. I'm OK with this approach if you don't have time for that however.
2. Can we add a bit more contrast to the text in the navbar (i.e. make it brighter)? 
![Screenshot from 2022-12-06 09-47-25](https://user-images.githubusercontent.com/10000651/205864604-e4e56655-4d72-4eaf-bd60-ad2541bd2250.png)